docs/data/material/components/snackbars/snackbars.md

```diff
@@ -132,3 +111,13 @@ export default function MyComponent() {
   );
 }
 ```
+
+## Anatomy
+
+The Snackbar component is composed of a root <div> that ... to complement:
+
+```html
+<div class="MuiSnackbar-root">
+  <!-- snackbar content goes here -->
+</div>
```

Comment:
Here's the basic anatomy - I'm a little confused about what's going on here, honestly 😅 not sure why this Paper is involved or why the content is wrapped in two layers of divs but I guess it is what it is!

```suggestion
<div role="presentation" class="MuiSnackbar-root>
  <div class="MuiPaper-root MuiSnackbarContent-root" role="alert">
    <div class="MuiSnackbarContent-message">
      <!-- Snackbar content goes here -->
    </div> 
  </div>
</div>
```